src/content/chapter0_basics/lesson02_unqualified_imports/en.html

```diff
@@ -13,3 +14,9 @@
   Generally it is best to use qualified imports, as this makes it clear where
   the function is defined, making the code easier to read.
 </p>
+<h2>Values and Types </h2>
+<p>
+  Values are imported in a manner similar to functions. However, the `type`
+  keyword must precede the name of an imported type. See here for 
+  [code showing types and values being imported](https://tour.gleam.run/standard-library/option-module/).
```

Comment:
This breaks the flow of the tour as it makes people jump forwards. All lessons should be self contained and build on previous lessons, so we'll have to show a type being imported.